homeassistant/components/hassio/issues.py

```diff
@@ -303,12 +306,17 @@ async def setup(self) -> None:
             self._hass, EVENT_SUPERVISOR_EVENT, self._supervisor_events_to_issues
         )
 
-    async def update(self) -> None:
+    async def update(self, _: datetime | None = None) -> None:
```

Comment:
Maybe it's too late, but I'd consider making this method protected to avoid outside usage that could spawn multiple retries in the same time span.

Reply:
Alternatively we could cancel any already scheduled retry before scheduling a new one.

Reply:
Hm yea good call. I'll make a PR for that

Reply:
https://github.com/home-assistant/core/pull/113425